tests/platform_tests/broadcom/test_ser.py

```diff
@@ -10,7 +10,7 @@
 
 pytestmark = [
     pytest.mark.disable_loganalyzer,
-    pytest.mark.broadcom,
+    pytest.mark.asic('broadcom'),
```

Comment:
I think it will be better to update this condition instead:
https://github.com/sonic-net/sonic-mgmt/blob/8b16ae88185658d2d537299e8f5b6f546a992596/tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml#L749

Reply:
@vperumal please address the comment

Reply:
Hi @Blueve / @abdosi , test_mark_conditions yaml file is more for skipping a script for a vendor. But this script is specifically written for Broadcom and not supposed to be run for any other vendor. By using mark.asic in a script, the script is run only for Broadcom asic. Similar skip is done for mellanox based scripts in https://github.com/sonic-net/sonic-mgmt/tree/master/tests/platform_tests/mellanox. All scripts there follow the same pytest.mark